### ISSUE TITLE
Conditional breakpoint settings will not close and save condition

### ISSUE THREAD

**Issue by stianborgesen** [gwomacks/php-debug#172]
If I right click at breakpoint, and open breakpoint settings, I can input my condition. Even multiple conditions. My problem occurs when I want to store the condition, and exit this view. I can't find a button for this. And clicking outside of the view does not work. 

I was however able to keep the conditional breakpoint by opening the developer tools in Atom and deleting the DOM element representing the breakpoint settings. 

Am I missing something, or should there be something like a button to close this view?

**Comment by willwh** [comment]
I ran in to this same issue, did you figure this out @stianborgesen ?

**Comment by cchamplin** [comment]
I will check into reproducing this.

**Comment by cchamplin** [comment]
Can you tell me what atom theme you are using? There should be an "x" in the top right corner of the breakpoint settings overlay.

**Comment by willwh** [comment]
@cchamplin sorry I've been using PHPStorm, but I would love to have this working well in Atom. I'm using the one-dark theme - https://github.com/atom/one-dark-ui

I do indeed see an "X" in the modal for setting breakpoint settings; however, it seems, once you hit the "X" -> the breakpoint gets removed altogether, making this feature useless :)

![image](https://cloud.githubusercontent.com/assets/398396/19420074/c67bcea8-9398-11e6-9218-7598e9022ce2.png)

Please let me know if there is any other information I can provide that would be helpful, I wrote a [blog post](https://www.northstudio.com/blog/debugging-drupal-8-atomio-xdebug-and-vagrant) about it, which I'd love to improve with conditional breakpoint instructions :)

**Comment by cchamplin** [comment]
Just to to double check, after you input your conditional are you clicking the "Add condition" before clicking the "X"? In retrospect I realize this is not especially intuitive.

**Comment by willwh** [comment]
Yes!

**Comment by venatiodecorus** [comment]
I'm running into the same issue, except when I click the X (after inputting my conditions and clicking "Add condition" the breakpoint disappears, but the Breakpoint list still lists a breakpoint but the line number changes to 1, and there is no visible breakpoint in the gutter.

**Comment by stephenharris** [comment]
I've getting this issue too (version 0.2.4). However, I also found that by leaving the "add condition" overlay open with the condition added, and running the application, then the condition _would_ apply. Closing the overlay once (if) the breakpoint was hit would remove the breakpoint, but the script would still be halted at that breakpoint.

So the breakpoint is lost when the overlay is closed.
